### PR TITLE
feature(OS-127): fe create modal language window mob

### PR DESCRIPTION
**Pull request by olenamshrv**

### CODE REVIEW

File: src/components/Selector/index.tsx [hunk 1]
```diff
@@ -14,12 +14,47 @@ export interface CountrySelectorProps {
...
+  const selectStyle: StylesConfig<SelectOption, IsMulti> = {
+    control: provided => ({
+      ...provided,
+      ...customControlStyles,
+    }),
+    menuList: provided => ({
+      ...provided,
+      ...customMenuListStyles,
+    }),
+  };
```
Comment (denred) [inline src/components/Selector/index.tsx:46]: looks like helper

File: src/components/HomePageModal/index.module.scss [hunk 2]
```diff
@@ -16,22 +16,62 @@
...
-  padding-bottom: 75px;
-  margin-bottom: 30px;
+  justify-content: flex-start;
+  gap: 20%;
+  padding-bottom: 77px;
+  margin-bottom: 32px;
   border-bottom: 1px solid $grayLight;
+  height: 100%;
+
+  @media only screen and (max-width: 650px) {
```
[the remark this answers is not in the document]
Comment (olenamshrv) [reply src/components/HomePageModal/index.module.scss:35]: height: 100%; - this is no longer necessary, will be removed

 @media only screen and (max-width: 650px) - for changing layout, at this breakpoint, the selector for location becomes too narrow

File: src/components/Selector/index.tsx [hunk 3]
```diff
@@ -14,12 +15,35 @@ export interface CountrySelectorProps {
...
-    <Select options={options} value={value} onChange={onChange} />
-  </div>
-);
+const Selector: FC<CountrySelectorProps> = ({
+  options,
+  value,
+  onChange,
+  isMobile,
+}) => {
+  const customControlStyles: CSSProperties = {
```
[the remark this answers is not in the document]
Comment (olenamshrv) [reply src/components/Selector/index.tsx:27]: Ready component Select from 'react-select' is used in Selector component. If just to add a wrapper to it with its own scss  it won't work. The styles changing works only through passing them to the corresponding internal components of Select, such as control, menuList etc. Maybe it's not the correct understanding from my side, please advise.

[the remark this answers is not in the document]
Comment (denred) [reply src/components/Selector/index.tsx:27]: If understand correctly, you need something like this:
```
const selectStyle = useCallback(
    (): StylesConfig<SelectOption, false> => ({
      control: provided => ({
        ...provided,
        className: styles.customControl,
      }),
      menuList: provided => ({
        ...provided,
        className: styles.customMenuList,
      }),
    }),
    [],
  );
```

**Pull request merged**